src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(new File(fromFileName)));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(new File(toFileName)))) {
+            int[] data = getDataFromFile(reader);
+            supply = data[0];
+            buy = data[1];
+            result = data[2];
+            writeDataToTheFile(writer, supply, buy, result);
+        } catch (IOException e) {
+            throw new RuntimeException("Something went wrong.");
+        }
+    }
+
+    public int[] getDataFromFile(BufferedReader reader) throws IOException {
+        int[] array = new int[3];
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
```

Comment:
let's not store it in a temp array, it's a fragile approach - when you change something here, you will need to change logic in the method that relies on this data too. Let's form a result string right in this method, this would also ensure compliance with SRP - writer utility method should only be concerned with writing of given string content, not forming it by itself

Reply:
I changed this method and it will return a String